### PR TITLE
Add use case for lowNodeUtilization

### LINKED ISSUE #212: Elements of nodeResourceUtilizationThresholds should be optional
The "balance" of my workload is solely based on Memory requests. I have no idea what my cpu or pod usages look like. And for the sake of the descheduler, I don't really care. How can I have it identify under and over utilized nodes based purely on Memory?

### PR DESCRIPTION
**Pull request by lixiang233**
Fixes #212

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -87,6 +87,41 @@ strategies:
...
+strategies:
+  "LowNodeUtilization":
+    enabled: true
+    params:
+      nodeResourceUtilizationThresholds:
+        thresholds:
+          "memory": 20
+        targetThresholds:
+          "memory": 70
+  "RemoveDuplicates":
```
[the remark this answers is not in the document]
Comment (ingvagabund) [reply docs/user-guide.md:111]: The same holds for the previous `Balance Cluster By Pod Age` example.

**Review by ingvagabund — COMMENTED** [review]
Nits

File: docs/user-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lixiang233) [reply docs/user-guide.md]: Fixed

**Comment by ingvagabund** [comment]
LGTM

**Review by damemi — APPROVED** [review]
/approve

**Pull request merged**